apps/ledger-live-desktop/src/renderer/screens/exchange/index.tsx

```diff
@@ -33,6 +42,28 @@ const LiveAppExchange = ({ appId }: { appId: string }) => {
   const manifest = localManifest || mockManifest || remoteManifest;
   const themeType = useTheme().colors.palette.type;
 
+  /**
+   * Pass correct account ID
+   * Due to Platform SDK account ID not being equivalent to Wallet API account ID
+   */
+  const customUrlParams = useMemo(() => {
+    if (
+      urlParams?.account &&
+      manifest?.apiVersion &&
+      semver.satisfies(WALLET_API_VERSION, manifest.apiVersion)
```

Comment:
I am not sure if the validation is the correct one.

semver.satisfies(manifest.apiVersion, "2.x" )

Reply:
tested and works ok, we can extend it later if needed.

Reply:
```tsx
      semver.satisfies(WALLET_API_VERSION, manifest.apiVersion)
```
This code is correct and `sermver.satisfies` allows for more versions to be checked thanks to ranges, this is why every wallet-api app is setup with `apiVersion: "^2.0.0"` meaning that if we update the constant `WALLET_API_VERSION` in LL, your app will still work

Reply:
If you want to have a manifest only for a specific wallet-api version it is possible if you omit the syntax for ranges like `^`, but this need more testing as we never had this use case before

Reply:
We can have a chat tomorrow and get a ticket created but will have to consider its priority.